examples/src/bin/importer.rs

```diff
@@ -55,7 +55,7 @@ async fn main() -> Result<()> {
                 if iroh_util::verify_hash(&cid, &data) == Some(false) {
                     bail!("invalid hash {:?}", cid);
                 }
-                let links = iroh_resolver::parse_links(&cid, &data).unwrap_or_default();
+                let links = iroh_content::util::parse_links(&cid, &data).unwrap_or_default();
```

Comment:
just reexport `parse_links` at the top level, no need for the `util` mod to be exposed